### PR TITLE
[openvpn] remove openvpn.storageClass from deckhouse configmap

### PR DESCRIPTION
**Pull request by vitaliy-sn**
## Description

* Temporarily returned the storageClass parameter in the spec.
* Added removal of openvpn.storageClass parameter from deckhouse configmap to migration hook.

## Why do we need it, and what problem does it solve?

The storageClass parameter is no longer needed.

## Changelog entries

```changes
section: openvpn
type: fix
summary: Removing of openvpn.storageClass parameter from deckhouse configmap
```

### CODE REVIEW

File: ee/fe/modules/500-openvpn/openapi/config-values.yaml [hunk 1]
```diff
@@ -205,3 +205,5 @@ properties:
           format: int64
         value:
           type: string
+  storageClass:
+    type: string
```
Comment (nabokihms) [inline ee/fe/modules/500-openvpn/openapi/config-values.yaml:209]: The `description` field is missing. This field will be shown in the docs, so we need to provide a description for it and mention that this field is deprecated.

File: ee/fe/modules/500-openvpn/hooks/easyrsa_migration.go [hunk 2]
```diff
@@ -95,6 +95,8 @@ func applyMigrationSecretFilter(obj *unstructured.Unstructured) (go_hook.FilterR
 }
 
 func migration(input *go_hook.HookInput) error {
+	input.ConfigValues.Remove("openvpn.storageClass")
```
Comment (nabokihms) [inline ee/fe/modules/500-openvpn/hooks/easyrsa_migration.go:98]: Could you please add a comment on why should we delete this field in the easyrsa migration?

**Review by nabokihms — CHANGES_REQUESTED** [review]

File: ee/fe/modules/500-openvpn/openapi/config-values.yaml (hunk 1 above)
Comment (vitaliy-sn) [reply ee/fe/modules/500-openvpn/openapi/config-values.yaml:209]: Wanted to do without it. Added a description.

File: ee/fe/modules/500-openvpn/hooks/easyrsa_migration.go (hunk 2 above)
Comment (vitaliy-sn) [reply ee/fe/modules/500-openvpn/hooks/easyrsa_migration.go:99]: Done

**Review by nabokihms — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.32

**Pull request merged**